backend/models/office_hours/event_type.py

```diff
@@ -12,16 +12,16 @@ class OfficeHoursEventType(Enum):
     Determines the type of an office hours event.
     """
 
-    OFFICE_HOURS = 0
-    TUTORING = 1
-    REVIEW_SESSION = 2
+    OFFICE_HOURS = "Office Hours"
+    TUTORING = "Tutoring"
+    REVIEW_SESSION = "Review Session"
 
 
 class OfficeHoursEventModeType(Enum):
     """
     Determines the office hours event mode.
     """
 
-    IN_PERSON = 0
-    VIRTUAL_STUDENT_LINK = 1
-    VIRTUAL_OUR_LINK = 2
+    IN_PERSON = "In-Person"
+    VIRTUAL_STUDENT_LINK = "Virtual - Student Link"
+    VIRTUAL_OUR_LINK = "Virtual - Our Link"
```

Comment:
Same concern here going from int to string. For now, I'd say let's revert. Perhaps at the model layer we can convert to string with a custom function that does the quick conversion, if the model is expecting a string for API endpoints.